packages/react-composites/tests/browser/chat/app/FakeAdapterApp.tsx

```diff
@@ -55,29 +115,50 @@ export const FakeAdapterApp = (): JSX.Element => {
   );
 };
 
-async function createFakeChatAdapter(model: ChatAdapterModel): Promise<ChatAdapter> {
-  const chatService = new FakeChatService();
+async function createFakeChatAdapter(
+  model: ChatAdapterModel,
+  hasRemoteFileSharingMessage: boolean
+): Promise<ChatAdapter> {
+  const m = new Model({ asyncDelivery: false });
```

Comment:
this creates a new Model called `m` but also takes in `model` as a function parameter - what is the difference between these two? (why do we create a new one here?0

Reply:
actually I don't see m used, was it left in from testing things?